changelog/19.0/19.0.0/summary.md

```diff
@@ -65,6 +66,13 @@ Prior to 19.0 VTTablet reported how much time non-streaming executions spend wai
 
 The build version (e.g., `19.0.0-SNAPSHOT`) has been added to `/debug/vars`, allowing users to programmatically inspect Vitess components' build version at runtime.
 
+#### <a id="vttablet-query-cache-hits-and-misses"/>VTTablet Query Cache Hits and Misses
+
+VTTablet exposes two counter stats:
+
+ * `QueryCacheHits`: Query engine query cache hits
+ * `QueryCacheMisses`: Query engine query cache misses
+
```

Comment:
This should go in the `changelog/20.0/20.0.0/summary.md` summary file, if it does not exist yet you can create it.

Reply:
Also a small nit picking, reading this it seems like VTTablet is only exposing these two counter stats now, maybe add "new" or something